templates/command-page.html

```diff
@@ -78,12 +78,54 @@
         <dd>{{ command_data_obj.since }}</dd>
     <dl>
     {% endif %}
-    {% if command_data_obj.acl_categories %}
+    {% if command_data_obj.acl_categories or command_data_obj.command_flags %}
```

Comment:
```suggestion
    {% if command_data_obj.acl_categories or command_data_obj.command_flags %}
```
What commands don't have command_flags?

Reply:
Module commands so JSON and bloom don't have them

Reply:
But they both already list full ACL categories as is